eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -8,6 +8,9 @@ parameters:
   - name: ArtifactName
     type: string
     default: ""
+  - name: TimeoutInMinutes
+    type: number
+    default: 90
```

Comment:
Do we need 90 mins to run? If not perhaps we should default to the usual 60 mins.

Reply:
We're timing out due to Gov Cloud not being able to deploy test resources within the standard timeout period.

Reply:
Changed to 60 as a default for consistency; we can now tune from the pipeline YML in the test directory.